### PR TITLE
bugfix(EWT-391): Adds alpha release channel for payment providers

### PR DESCRIPTION
**Pull request by dili91**
# Description

The library was missing the alpha release channel. We realised last week only with the help of a failed acceptance tests, once we release the new commercial VRP mock bank.

## Type of change

Please select multiple options if required.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code where necessary
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have updated the relevant documentation

### CODE REVIEW

File: src/test/java/com/truelayer/java/acceptance/MandatesAcceptanceTests.java [hunk 1]
```diff
@@ -111,7 +111,7 @@ public void itShouldCreateAMandateWithUserSelectedProvider() {
         // create mandate
         ProviderSelection userSelectedProvider = ProviderSelection.userSelected()
                 .filter(ProviderFilter.builder()
-                        .releaseChannel(ReleaseChannel.PRIVATE_BETA)
+                        .releaseChannel(ReleaseChannel.GENERAL_AVAILABILITY)
```
Comment (dili91) [inline src/test/java/com/truelayer/java/acceptance/MandatesAcceptanceTests.java:114]: We're no more in private beta

File: build.gradle [hunk 2]
```diff
@@ -78,7 +78,8 @@ tasks.register('acceptance-tests', Test) {
     useJUnitPlatform{
         includeTags "acceptance"
     }
-    testLogging.showStandardStreams = true
+    // toggle to see the HTTP traces when running acceptance tests
+    testLogging.showStandardStreams = false
```
Comment (dili91) [inline build.gradle:82]: Having these logs enabled was challenging readability at CI most of the time IMO

**Review by tl-luca-baggi — APPROVED** [review]

**Comment by dili91** [comment]
Currenlty on hold

**Comment by dili91** [comment]
It seems we do not intend to expose providers in alpha, so I'm closing this for now.

**Pull request closed without merging**